lib/addons/p5.dom.js

```diff
@@ -698,8 +702,19 @@
     else if (radios.length == 1){
       count = 1;
     }
-    var elt = document.createElement('div');
-    var self = addElement(elt, this);
+    // see if we got an existing set of radios from callee
+    var elt, self;
+    var arg = arguments[0];
```

Comment:
in the spirit of PR #2424, can we add a named argument to the signature and test it's type instead of extracting arguments[0]?